packages/api/src/external/ehr/athenahealth/shared.ts

```diff
@@ -33,7 +33,7 @@ export function createMetriportAddresses(patient: PatientResource): Address[] {
       addressLine2: address.line.length > 1 ? address.line.slice(1).join(" ") : undefined,
       city: address.city,
       state: normalizeUSStateForAddress(address.state),
-      zip: normalizeZipCode(address.postalCode),
+      zip: normalizeZipCodeNew(address.postalCode),
```

Comment:
Nice! Just to confirm, there are still other usages of `normalizeZipCode`, right? If not, we could also remove it.

Reply:
Was also used in MPI, but I've updated those as well.

This PR https://github.com/metriport/metriport/pull/2953/files addresses a full alignment between MPI / demo aug / patient import / etc.